### PR TITLE
Articles/prism/mocking.md

### LINKED ISSUE #64: articles/prism/mocking.md

### PR DESCRIPTION
**Pull request by pytlesk4**
closes #64

### CODE REVIEW

File: articles/prism/mocking.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+}
+```
+5. **Save** the OAS file.
+
+![](../../assets/gifs/mock-get-hello-200.gif)
+
+6. Create a new **Prism instance file** in the project. Name it `hello-mock.prism`.
+7.  Add an API to the Prism instance and connect the `hello.oas2` specification that you created earlier.
+
+<callout> Prism instances are made up of APIs and Rules, you can learn more about them here. </>
```
Comment (rowa97) [inline articles/prism/mocking.md]: missing link

File: articles/prism/mocking.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+8. Add a **new rule** that you will setup to power mocking. 
+
+<callout> Rules simply apply scenarios to HTTP traffic passing through the Prism instance. </>
+
+9. Connect the **rule** to the API by clicking on the  `apis` dropdown input and selecting the previously created API.   . 
+10. Add a **scenario** that will perform the mocking. We have an official Stoplight mock scenario [here](https://next.stoplight.io/stoplight/prism?edit=%23%2Fscenarios%2Fmock), which makes it easy to get started.
+    1. Add a scenario to the `before` section of your rule.
+    2. Select `another project` in the first dropdown and  search for `prism`. 
+    3. Locate the file `helpers.prism.yml` and select the scenario `mock`.
```
Comment (rowa97) [inline articles/prism/mocking.md]: I wasn't sure what the action was here so I chose select. Could use a double check here.

**Comment by marbemac** [comment]
Putting a hold on this until stoplightio/core#707 is completed. We can then update the steps to be mucchhh simpler.

**Comment by pytlesk4** [comment]
@marbemac what should we do about this article?

**Comment by tbarn** [comment]
I feel like steps #1-5 could be summed up, by referring to another doc or blog post about how to create a spec. Otherwise, the only other updates it really needs are just new GIFs or in some cases just screenshots for everything else that's changed since earlier this year.

**Comment by marbemac** [comment]
@pytlesk4 were you able to get to this yesterday?

**Comment by pytlesk4** [comment]
Going to get to this tonight, I just need to recreate the gifs.

**Comment by pytlesk4** [comment]
@marbemac / @tbarn updates pushed, please review. I didn't remove steps 1-5 because the whole tutorial was based on creating specific operations. I added a link to the modeling introduction.

Let me know if I need to add anything else.

**Comment by tbarn** [comment]
I'd recommend a suggestion near the start of the steps to jump to the step 6 for anyone who has already created their OAS file. Otherwise, you'll lose people thinking it isn't a tutorial for them. 

It appears like you are missing some links. Robbins commented on one, but I noticed another random `</>`

**Comment by pytlesk4** [comment]
@tbarn I think the </> is for a callout, going to finish this up today though.

**Comment by pytlesk4** [comment]
- updated

**Comment by pytlesk4** [comment]
@tbarn nay/yay?

**Comment by pytlesk4** [comment]
@tbarn can we release this?

**Comment by pytlesk4** [comment]
@rowa97 💯

**Pull request merged**